mito-ai/src/Extensions/SettingsManager/database/model.ts

```diff
@@ -24,10 +24,60 @@ export interface DatabaseField {
 export interface DatabaseConfig {
     type: string;
     displayName: string;
+    alertText?: string;  // Optional HTML content for alerts/notifications
     fields: DatabaseField[];
 }
 
 export const databaseConfigs: Record<string, DatabaseConfig> = {
+    mssql: {
+        type: 'mssql',
+        displayName: 'Microsoft SQL Server',
+        alertText: 'Microsoft SQL Server requires an additional driver to be installed. For more info, consult the <a href="https://docs.trymito.io/mito-ai/database-connectors/microsoft-sql-server" target="_blank">Mito docs</a>.',
```

Comment:
Once I installed the driver, I was still surprised to see this alert. I don't think we need to add logic now to detect if its there or not, but maybe we can just update the messaging slightly to let users know that they might already have it???